docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/braze-tag-for-gtm-ss/braze-tag-configuration/index.md

```diff
@@ -14,12 +14,72 @@ Set this to the URL of your Braze REST [endpoint](https://www.braze.com/docs/api
 
 Set this to your Braze [API Key](https://www.braze.com/docs/api/basics/#app-group-rest-api-keys) that will be included in each request. The minimum permission that you need to assign for this API Key is to access the `/users/track` endpoint.
 
-![](images/key_permission.png)
+![key permission](images/key_permission.png)
 
-### Braze external\_id (required)
+### Identity settings
+
+_Note: Since v0.2.0 this section of the configuration has changed. Below you can read about the latest configuration options. If you are interested about pre-0.2.0 versions, there is a section at the end of this one._
```

Comment:
I think you can drop this note and simply update this to v0.2.0 as GTM tags are always latest in reality.